src/app/choose-template/page.tsx

```diff
@@ -0,0 +1,42 @@
+'use client' // we need this to be a client component because nhsuk-react-components uses client-only react features
+import { Radios, Button, ErrorSummary } from 'nhsuk-react-components';
+import { useFormState } from 'react-dom';
+import { chooseTemplateServerAction, FormState } from '../../utils/choose-template-server-action';
+
+export default function ChooseTemplatePage() {
+    const [state, action] = useFormState<FormState, FormData>(chooseTemplateServerAction, {});
+
+    return (
+        <>
+            {state?.validationError ? (
+            <div>
+                <ErrorSummary>
+                    <ErrorSummary.Title>
+                        {state?.validationError}
+                    </ErrorSummary.Title>
+                </ErrorSummary>
+            </div>
+            ) : undefined}
+            <h2 className='nhsuk-heading-l' data-testid='page-sub-heading'>
+                Choose a template type to create
+            </h2>
+            <form action={action}>
+                <Radios id="choose-template-radios">
```

Comment:
Can we tie the component in to the validation error, I think that there is a way to make it red if it is the offending field.

Reply:
updated